### PR TITLE
Add neotest utility.

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**
This adds the utility that can fill and read the DLS node using
neo protocol.

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
It's probably time to start thinking about actual neo CLI clients.

File: neotest/main.d [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                return;
+            }
+        default:
+            assert(false);
+        }
+    }
+
+    abstract protected void go ( );
+
+    protected void putNotifier ( DlsClient.Neo.Put.Notification info,
```
Comment (gavin-norman-sociomantic) [inline neotest/main.d]: Seems this is unused.

File: neotest/main.d [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+                    default: assert(false);
+                }
+                break;
+
+            default: assert(false);
+        }
+    }
+
+    protected void getRangeNotifier ( DlsClient.Neo.GetRange.Notification info,
```
Comment (gavin-norman-sociomantic) [inline neotest/main.d]: Would probably make more sense to put this inside `GetRangeTest`.

**Comment by nemanja-boric-sociomantic** [comment]
Updated.

**Comment by nemanja-boric-sociomantic** [comment]
> It's probably time to start thinking about actual neo CLI clients.

Yes, unfortunately, recent events showed that turtle tests are not enough to test the node for the production state.

**Review by gavin-norman-sociomantic — APPROVED** [review]

**Pull request merged**